code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -130,3 +130,8 @@
 		for(var/obj/item/weapon/bomberman/dispenser in src)
 			if(dispenser.spam_bomb)
 				dispenser.attack_self(src)
+
+		if(get_active_hand() && istype(get_active_hand(),/obj/item/weapon/rcl))

```

Comment:
Will this cause problems if get_active_hand is not an RCL? For instance if you're saying...

var/obj/item/weapon/rcl/R = /obj/item/reagent_containers/glass/beaker ?